tests/saml/configuration/test_model.py

```diff
@@ -503,7 +504,6 @@ def test_get_settings_returns_correct_result(self):
                 "logoutResponseSigned": False,
                 "wantMessagesSigned": False,
                 "metadataCacheDuration": None,
-                "rejectUnsolicitedResponsesWithInResponseTo": False,
```

Comment:
I don't understand why Python 3 conversion by itself would make these changes.

Reply:
It didn't. This property is no longer in the object.